db/create-jade-db

```diff
@@ -0,0 +1,3 @@
+CREATE USER jade WITH PASSWORD 'jadepasswd';
```

Comment:
I suggest avoiding use of the project name in the code. In my experience, project names often need to be changed. For example, another open source project or commercial product named Jade that we collide with.

My preference would be to name the user "data-repo-manager" and the database "data-repo-metadata" or similar.